draft-ietf-quic-transport.md

```diff
@@ -2992,9 +2993,8 @@ valid frames? -->
 ## Generating Acknowledgements {#generating-acks}
 
 Endpoints acknowledge all packets they receive and process. However, only
-ack-eliciting packets (see {{QUIC-RECOVERY}}) trigger the sending of an ACK
-frame.  Packets that are not ack-eliciting are only acknowledged when an ACK
-frame is sent for other reasons.
+ack-eliciting packets trigger the sending of an ACK frame.  Packets that are not
```

Comment:
Can we fix this here?  I know that this isn't this PR, but I would prefer to say:

```suggestion
ack-eliciting packets cause an ACK frame to be sent within the maximum ack delay.  Packets that are not
```

Or something like that.